fix: security issue

Faraz32123 opened the pull request:
- Attaching error for context:
`Uncaught SecurityError: Failed to read a named property 'offsetHeight' from 'Window': Blocked a frame with origin "https://studio.stage.edx.org/" from accessing a cross-origin frame.`
- It's related previous PR: https://github.com/openedx/edx-platform/pull/36643/files.
- This time followed [LMS template](https://github.com/openedx/edx-platform/blob/5b3caa93e218e38e7459bdc1ff99ee175e93a22b/lms/templates/courseware/courseware-chromeless.html#L126C7-L127C42) as we are not getting any error on LMS.

navinkarkera commented on common/templates/xblock_v2/xblock_iframe.html:
```diff
@@ -455,6 +455,9 @@
             if (window !== window.parent) {
+                var lastHeight = window.offsetHeight;
+                var lastWidth = window.offsetWidth;
+
```
We don't need these lines, they were removed by https://github.com/openedx/edx-platform/pull/36643

Faraz32123 replied:
Are we sure as [LMS template](https://github.com/openedx/edx-platform/blob/5b3caa93e218e38e7459bdc1ff99ee175e93a22b/lms/templates/courseware/courseware-chromeless.html#L126C7-L127C42) has these lines.

Faraz32123 replied:
Let me remove these, We can match the other file and see if that work for us.

navinkarkera replied:
> Are we sure as [LMS template](https://github.com/openedx/edx-platform/blob/5b3caa93e218e38e7459bdc1ff99ee175e93a22b/lms/templates/courseware/courseware-chromeless.html#L126C7-L127C42) has these lines.

Yes, we are not using them here.

navinkarkera commented on common/templates/xblock_v2/xblock_iframe.html:
```diff
@@ -469,6 +472,10 @@
+
+                    lastHeight = newHeight;
+                    lastWidth = newWidth;
+
```
Same here: https://github.com/openedx/edx-platform/pull/36643

Commit:
fix: security issue

- It's related previous PR: https://github.com/openedx/edx-platform/pull/36643/files.
- This time followed LMS template(https://github.com/openedx/edx-platform/blob/5b3caa93e218e38e7459bdc1ff99ee175e93a22b/lms/templates/courseware/courseware-chromeless.html#L126C7-L127C42) as we are not getting any error on LMS.

Attaching error for context:
`Uncaught SecurityError: Failed to read a named property 'offsetHeight' from 'Window': Blocked a frame with origin "https://studio.stage.edx.org/" from accessing a cross-origin frame.`

rayzhou-bit commented:
@navinkarkera Could you please review again? I think merging is blocked due to requested changes.
